libfreerdp/codec/dsp.c

```diff
@@ -261,10 +268,6 @@ static BOOL freerdp_dsp_resample(FREERDP_DSP_CONTEXT* context, const BYTE* src,
 	*length = Stream_Length(context->resample);
 	return (error == 0) ? TRUE : FALSE;
 #else
-	WINPR_UNUSED(src);
-	WINPR_UNUSED(size);
-	WINPR_UNUSED(data);
-	WINPR_UNUSED(length);
```

Comment:
Why remove these?

Reply:
Because they are now always used. I have added:

```
		*data = src;
		*length = size;
```

...in the "Fix dsp resemple" commit.